.golangci.yml

```diff
@@ -4,13 +4,90 @@ run:
 #   timeout: 5m
 
 linters:
+  # Disable all linters.
+  # Default: false
   disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
+    - asciicheck
+    - bidichk
+    - bodyclose
+    - contextcheck
+    - cyclop
+    - deadcode
+    - depguard
+    - dogsled
```

Comment:
This one seems a bit odd to me

Reply:
doesn't it?

ps we are now talking via vscode.  it is neat.